src/test/rustdoc/reexports-priv.rs

```diff
@@ -0,0 +1,48 @@
+// aux-build: reexports.rs
+// compile-flags: --document-private-items
+
+#![crate_name = "foo"]
+
+extern crate reexports;
+
+// @has 'foo/macro.addr_of.html' '//*[@class="docblock type-decl"]' 'pub macro addr_of($place : expr) {'
+pub use reexports::addr_of;
+// @has 'foo/macro.addr_of_crate.html' '//*[@class="docblock type-decl"]' 'pub(crate) macro addr_of_crate($place : expr) {'
+pub(crate) use reexports::addr_of_crate;
+// @has 'foo/macro.addr_of_self.html' '//*[@class="docblock type-decl"]' 'pub(crate) macro addr_of_self($place : expr) {'
```

Comment:
Maybe add a test with multiple matchers for completeness?

Reply:
What do you mean?

Reply:
Ah sorry. Meant to link [here](https://github.com/rust-lang/rust/pull/86841/files#diff-9ae3607a55a104c791e37441a1f68b776f76cbb9ec02b73fc22b1f75693fb3ecR3), but I mean't to add a macro that contains multiple matchers or is this covered somewhere else?

Reply:
Oh I see! Yes it's tested in other places. I didn't change this part, only the macros "header".